sdk/typescript/test/unit/zklogin/signature.test.ts

```diff
@@ -60,4 +64,16 @@ describe('zkLogin signature', () => {
 			}),
 		).toBe(aSignature);
 	});
+	it('verify a personal message runs', async () => {
+		// test case generated from `sui keytool zk-login-insecure-sign-personal-message --data "hello"`
+		let testSignature =
+			'BQNNMTczMTgwODkxMjU5NTI0MjE3MzYzNDIyNjM3MTc5MzI3MTk0Mzc3MTc4NDQyODI0MTAxODc5NTc5ODQ3NTE5Mzk5NDI4OTgyNTEyNTBNMTEzNzM5NjY2NDU0NjkxMjI1ODIwNzQwODIyOTU5ODUzODgyNTg4NDA2ODE2MTgyNjg1OTM5NzY2OTczMjU4OTIyODA5MTU2ODEyMDcBMQMCTDU5Mzk4NzExNDczNDg4MzQ5OTczNjE3MjAxMjIyMzg5ODAxNzcxNTIzMDMyNzQzMTEwNDcyNDk5MDU5NDIzODQ5MTU3Njg2OTA4OTVMNDUzMzU2ODI3MTEzNDc4NTI3ODczMTIzNDU3MDM2MTQ4MjY1MTk5Njc0MDc5MTg4ODI4NTg2NDk2Njg4NDAzMjcxNzA0OTgxMTcwOAJNMTA1NjQzODcyODUwNzE1NTU0Njk3NTM5OTA2NjE0MTA4NDAxMTg2MzU5MjU0NjY1OTcwMzcwMTgwNTg3NzAwNDEzNDc1MTg0NjEzNjhNMTI1OTczMjM1NDcyNzc1NzkxNDQ2OTg0OTYzNzIyNDI2MTUzNjgwODU4MDEzMTMzNDMxNTU3MzU1MTEzMzAwMDM4ODQ3Njc5NTc4NTQCATEBMANNMTU3OTE1ODk0NzI1NTY4MjYyNjMyMzE2NDQ3Mjg4NzMzMzc2MjkwMTUyNjk5ODQ2OTk0MDQwNzM2MjM2MDMzNTI1Mzc2Nzg4MTMxNzFMNDU0Nzg2NjQ5OTI0ODg4MTQ0OTY3NjE2MTE1ODAyNDc0ODA2MDQ4NTM3MzI1MDAyOTQyMzkwNDExMzAxNzQyMjUzOTAzNzE2MjUyNwExMXdpYVhOeklqb2lhSFIwY0hNNkx5OXBaQzUwZDJsMFkyZ3VkSFl2YjJGMWRHZ3lJaXcCMmV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNJc0ltdHBaQ0k2SWpFaWZRTTIwNzk0Nzg4NTU5NjIwNjY5NTk2MjA2NDU3MDIyOTY2MTc2OTg2Njg4NzI3ODc2MTI4MjIzNjI4MTEzOTE2MzgwOTI3NTAyNzM3OTExCgAAAAAAAABhAPl6x1AzJDGj7MEyBbKiIMd8l+o+yL1NU/9cs8OJIysI4Cy9GbT0GY3NnY1+0PaRaAXTm5EmeiWeHqYjn9OauAa5xu4WMO8+cRFEpkjbBruyKE9ydM++5T/87lA8waSSAA==';
+		let bytes = 'BWhlbGxv';
+		let parsed = parseSerializedZkLoginSignature(testSignature);
+		let pk = new ZkLoginPublicIdentifier(parsed.publicKey);
+
+		// returns error bc the test signature is for max epoch 10, should not verify.
+		let res = await pk.verifyPersonalMessage(fromB64(bytes), parsed.signature);
```

Comment:
this will hit mainnet, does this method work in localnet?  We currently don't have any other tests that make requests to services other than the localnet instance we are running for tests

Reply:
i dont know if theres a graphql client being spinned up during the test. i think hitting mainnet is ok unless there are other concerns. the endpoint should be stable